path_0: [0, 0,
0.399998, 0.0012636,
0.799996, 0.0023912,
1.14114, -0.0896726,
1.4138, -0.000795692,
1.71215, 0.224199,
2.16025, 0.165594,
2.4595, 0.244111,
2.93896, 0.328047,
3.29788, 0.483234,
3.44501, 0.798261,
3.8248, 0.921107,
4.12247, 1.18761,
4.39375, 1.48011,
4.67526, 1.76211,
4.96719, 1.98518,
5.31186, 2.17867,
5.64986, 2.39137,
5.9549, 2.70558,
6.34433, 2.79417,
6.35879, 3.14253,
6.26803, 3.41499,
6.2124, 3.67305,
6.37857, 3.9983,
6.2019, 4.21265,
6.12009, 4.69516,
5.84001, 4.99054,
5.47647, 5.12277,
5.32772, 5.55053
]
path_1: [5, 6,
4.6, 6.00124,
4.2244, 5.80274,
3.91977, 5.78263,
3.62342, 5.62293,
3.58806, 5.40997,
3.59655, 5.17097,
4.06908, 5.06268,
4.42819, 4.87354,
4.708, 4.56937,
4.69839, 4.22739,
4.92121, 3.98961,
5.02025, 3.62309,
5.4471, 3.58951,
5.37218, 3.42545,
5.00624, 3.1796,
4.68377, 2.98448,
4.47372, 2.61136,
4.89343, 2.48273,
5.06652, 2.18835,
5.2898, 1.92829,
5.56941, 1.90341,
5.76871, 1.58434,
5.85956, 1.25271,
5.99151, 1.00172,
5.76315, 0.741567,
5.43419, 0.539619,
5.08179, 0.307613,
4.6818, 0.3101,
4.26568, 0.126047,
3.88952, 0.110133,
3.48952, 0.111277,
3.08952, 0.11226,
2.68952, 0.112755,
2.28955, 0.116524,
1.88956, 0.117549,
1.53189, 0.0992592,
1.34197, -0.0468088,
0.944085, -0.207771,
0.544089, -0.206192,
0.367546, -0.459623,
-0.0407614, -0.654086,
-0.44944, -0.711901,
-0.652575, -0.977718,
-0.938902, -1.29335,
-1.27812, -1.14416,
-1.36451, -0.692724,
-1.69082, -0.496476,
-1.74118, -0.0997412,
-1.81456, 0.292506,
-1.81198, 0.692281,
-1.88288, 1.08445,
-2.0985, 1.2932,
-2.10401, 1.69066,
-2.13454, 2.08805,
-2.18921, 2.48074,
-2.24402, 2.87595,
-2.23826, 3.27108,
-2.32688, 3.6602,
-2.35984, 4.05506,
-2.39805, 4.45314,
-2.45541, 4.84811,
-2.50623, 5.24325,
-2.61552, 5.62681,
-2.93491, 5.80214,
-3.23418, 5.59152,
-3.47997, 5.83796,
-3.85381, 5.94648,
-4.17106, 6.13759,
-4.45927, 6.28458,
-4.87744, 6.13693,
-5.15392, 6.13912,
-5.3302, 5.89757
]
path_2: [-5.5, 5.5,
-5.3, 5.50031,
-4.9, 5.50086,
-4.5, 5.50144,
-4.15703, 5.54591,
-3.83115, 5.65542,
-3.36874, 5.68251,
-3.03555, 5.62964,
-2.60993, 5.52186,
-2.11727, 5.49663,
-1.9126, 5.32908,
-1.42179, 5.26563,
-1.36781, 4.85307,
-1.21887, 4.68928,
-0.874579, 4.56894,
-0.928131, 4.15714,
-0.967318, 3.98589,
-0.846484, 3.70271,
-0.775679, 3.38493,
-0.845512, 2.99042,
-0.732938, 2.84773,
-0.442138, 2.72971,
-0.065073, 2.59629,
-0.0117264, 2.34538,
-0.40891, 2.25665,
-0.868422, 2.16023,
-1.11898, 1.86821,
-1.50573, 1.7186,
-1.77962, 1.475,
-1.86295, 1.17176,
-1.43117, 1.01747,
-1.1523, 0.701386,
-0.78106, 0.516167,
-0.478985, 0.421398
]
path_3: [5, 6,
4.6, 6.00124,
4.2244, 5.80274,
3.91977, 5.78263,
3.62342, 5.62293,
3.58806, 5.40997,
3.59655, 5.17097,
4.06908, 5.06268,
4.42819, 4.87354,
4.708, 4.56937,
4.69839, 4.22739,
4.92121, 3.98961,
5.02025, 3.62309,
5.4471, 3.58951,
5.37218, 3.42545,
5.00624, 3.1796,
4.68377, 2.98448,
4.47372, 2.61136,
4.89343, 2.48273,
5.06652, 2.18835,
5.2898, 1.92829,
5.56941, 1.90341,
5.76871, 1.58434,
5.85956, 1.25271,
5.99151, 1.00172,
5.76315, 0.741567,
5.43419, 0.539619,
5.08179, 0.307613,
4.6818, 0.3101,
4.26568, 0.126047,
3.88952, 0.110133,
3.48952, 0.111277,
3.08952, 0.11226,
2.68952, 0.112755,
2.28955, 0.116524,
1.88956, 0.117549,
1.53189, 0.0992592,
1.34197, -0.0468088,
0.944085, -0.207771,
0.544089, -0.206192,
0.144877, -0.186772,
-0.245948, -0.257191,
-0.519796, -0.453408,
-0.409894, -0.819806,
-0.782163, -1.06027,
-0.980104, -1.48555,
-1.38006, -1.4805,
-1.66002, -1.71584,
-1.34043, -1.94567,
-1.1217, -2.26464,
-1.3329, -2.52467,
-1.63538, -2.76701,
-1.90444, -2.8273,
-2.1623, -3.05682,
-2.55671, -3.18474,
-2.43689, -3.51355,
-2.49527, -3.95144,
-2.5084, -4.36771,
-2.36634, -4.55441,
-2.35359, -4.66134,
-2.45176, -4.93753,
-2.802, -4.9862,
-3.20806, -5.01328,
-3.60722, -4.98926,
-3.93354, -4.6976,
-4.2239, -4.42405,
-4.43984, -4.55925,
-4.85571, -4.67945,
-5.04814, -4.55911
]
path_4: [-5.5, -5,
-5.30504, -4.95537,
-4.90578, -5.04861,
-4.506, -5.03915,
-4.24207, -5.20154,
-3.96728, -4.87332,
-3.51363, -4.68659,
-3.13827, -4.65674,
-2.77618, -4.62247,
-2.54458, -4.37033,
-2.46507, -3.98916,
-2.36697, -3.61491,
-2.29254, -3.22384,
-2.23066, -2.84949,
-2.05056, -2.50141,
-1.82929, -2.16824,
-1.5976, -1.84301,
-1.50101, -1.45692,
-1.4689, -1.05986,
-1.18992, -0.774956,
-1.01961, -0.572776,
-0.802655, -0.253114,
-0.66685, -0.0496648,
-0.427402, 0.270484
]
path_5: [5, 6,
4.77749, 5.80677,
5.15669, 5.64221,
5.55667, 5.64595,
5.41625, 5.44255,
5.05345, 5.23547,
4.77524, 4.92183,
4.41949, 4.76577,
4.41251, 4.66343,
4.47711, 4.25691,
4.4406, 4.09404,
4.15486, 3.73598,
3.75504, 3.74734,
3.6242, 3.44679,
3.97926, 3.27928,
4.37925, 3.2814,
4.74796, 3.18795,
4.97502, 2.77592,
4.98668, 2.47148,
5.2087, 2.17071,
5.38207, 1.88432,
5.35348, 1.47966,
5.3012, 1.18363,
5.15727, 0.886439,
4.81005, 0.737353,
4.41006, 0.740415,
4.02938, 0.63202,
3.70498, 0.640397,
3.38049, 0.436544,
3.23882, 0.362948,
3.20166, 0.126703,
2.80167, 0.127443,
2.40167, 0.128204,
2.00167, 0.128688,
1.60167, 0.129425,
1.20168, 0.132378,
0.801689, 0.134746,
0.34018, 0.0191664
]
